### DISCUSSION TITLE
Hide/show empty groups, remember last setting

### DISCUSSION THREAD

**Discussion by kbradley** [six7/figma-tokens#454] [Ideas]
I'm finding that every time I close the Plugin dialog and then reopen later it doesn't remember my last setting to "Hide empty groups" so I have to set it again each time to eliminate visual clutter. It would be great if the plugin would remember the last setting you used for this.

**Comment by six7** [comment]
Should be easy to add this to the list of settings we store for each user on their local storage, will work on this 👍

**Comment by six7** [comment]
Shipped in version 85!

**Reply by kbradley** [reply]
Thank you!
